multimedia/tvheadend/Makefile

```diff
@@ -26,120 +22,215 @@ include $(INCLUDE_DIR)/package.mk
+ifeq ($(CONFIG_TVHEADEND_NEWCAMD),)
+	CONFIGURE_ARGS += --disable-cwc
 endif
-
-ifeq ($(CONFIG_TVHEADEND_CCW_SUPPORT),)
-  CONFIGURE_ARGS += --disable-constcw
+ifeq ($(CONFIG_TVHEADEND_CAPMT),)
+	CONFIGURE_ARGS += --disable-capmt
 endif
-
-# Other options
-ifeq ($(CONFIG_TVHEADEND_AVAHI_SUPPORT),)
-  CONFIGURE_ARGS += --disable-avahi
-else
-  CONFIGURE_ARGS += --enable-avahi
+ifeq ($(CONFIG_TVHEADEND_CCCAM),)
+	CONFIGURE_ARGS += --disable-cccam
 endif
-
-ifeq ($(CONFIG_TVHEADEND_IMAGECACHE),)
-  CONFIGURE_ARGS += --disable-imagecache
-else
-  CONFIGURE_ARGS += --enable-imagecache
 endif
-
-ifeq ($(CONFIG_TVHEADEND_TRACE),)
-  CONFIGURE_ARGS += --disable-trace
 endif
 
-# libav and ffmpeg are broken, so remove codecs too.
+## Transcoding | Uncomment these.
+##ifeq ($(CONFIG_BUILD_PATENTED),)
+##	CONFIGURE_ARGS += --disable-libav --disable-libx264 --disable-libx265 --disable-libx265_static
+##else
+##ifeq ($(CONFIG_TVHEADEND_LIBFFMPEG),)
+##	CONFIGURE_ARGS += --disable-libav
+##else
+##	CONFIGURE_ARGS += --enable-libav
+##endif
+##ifeq ($(CONFIG_TVHEADEND_LIBX264),)
+##	CONFIGURE_ARGS += --disable-libx264
+##endif
+##ifeq ($(CONFIG_TVHEADEND_LIBX265),)
+##	CONFIGURE_ARGS += --disable-libx265 --disable-libx265_static
+##endif
+##ifneq ($(CONFIG_TVHEADEND_LIBFDKAAC),)
+##	CONFIGURE_ARGS += --enable-libfdkaac
+##endif
+##endif
+##
+##ifeq ($(CONFIG_TVHEADEND_LIBVPX),)
+##	CONFIGURE_ARGS += --disable-libvpx
+##endif
+##
+##ifeq ($(CONFIG_TVHEADEND_LIBlibtheora),)
+##	CONFIGURE_ARGS += --disable-libtheora
+##endif
+##
+##ifeq ($(CONFIG_TVHEADEND_LIBVORBIS),)
+##	CONFIGURE_ARGS += --disable-libvorbis
+##endif
+##
+##ifeq ($(CONFIG_TVHEADEND_LIBOPUS),)
+##	CONFIGURE_ARGS += --disable-libopus
+##endif
+
 CONFIGURE_ARGS += \
 	--arch=$(ARCH) \
+	--disable-libsystemd_daemon \
 	--disable-dbus_1 \
 	--disable-libav \
 	--disable-ffmpeg_static \
-	--disable-libx264 \
-	--disable-libx264_static \
 	--disable-libx265 \
 	--disable-libx265_static \
+	--disable-libx264 \
+	--disable-libx264_static \
 	--disable-libvpx \
 	--disable-libvpx_static \
 	--disable-libtheora \
 	--disable-libtheora_static \
 	--disable-libvorbis \
 	--disable-libvorbis_static \
+	--disable-libopus \
+	--disable-libopus_static \
 	--disable-libfdkaac \
 	--disable-libfdkaac_static \
+	--disable-pcloud_cache \
 	--enable-bundle \
 	--nowerror=unused-variable
 
+## Transcoding | Remove these from CONFIGURE_ARGS.
+##	--disable-libav \
+##	--disable-libx265 \
+##	--disable-libx265_static \
+##	--disable-libx264_static \
+##	--disable-libvpx_static \
+##	--disable-libtheora_static \
+##	--disable-libvorbis_static \
+##	--disable-libopus_static \
+##	--disable-libfdkaac_static \
+##	--disable-pcloud_cache \
+
 define Build/Prepare
 	$(call Build/Prepare/Default)
 	echo 'Tvheadend $(shell echo $(PKG_SOURCE_VERSION) | sed "s/^v//")~openwrt$(PKG_RELEASE)' \
 		> $(PKG_BUILD_DIR)/debian/changelog
 endef
 
 define Package/conffiles
-/etc/config/tvheadend
+	/etc/config/tvheadend
```

Comment:
This is wrong. Did you tested it? It would not be marked as configuration file anymore.